### PR TITLE
Fix misat's grammer

### PR DESCRIPTION
**Pull request by linusstretchtips**
misat's grammer is shit

### CODE REVIEW

**Comment by linusstretchtips** [comment]
added more grammer shit

File: plugin/src/main/java/org/screamingsandals/bedwars/Main.java [hunk 1]
```diff
@@ -553,7 +553,7 @@ public void enable() {
         /* Initialize our ScoreboardLib*/
         ScoreboardManager.init(this.getPluginDescription().as(JavaPlugin.class));
 
-        PlayerMapper.getConsoleSender().sendMessage(Component.text("Everything is loaded! If you like our work, consider visiting our Patreon! <3").color(NamedTextColor.WHITE));
+        PlayerMapper.getConsoleSender().sendMessage(Component.text("everything has finished loading! If you like our work, consider subscribing to our Patreon! <3").color(NamedTextColor.WHITE));
```
Comment (Misat11) [inline plugin/src/main/java/org/screamingsandals/bedwars/Main.java:556]: It's start of a sentence so big E please xdd. And consider checking the base English language file, this file is not updated via Crowdin (don't update keys, only values tho)

**Review by Misat11 — CHANGES_REQUESTED** [review]

**Comment by Misat11** [comment]
And btw, grammar is written with a, not e.

File: plugin/src/main/java/org/screamingsandals/bedwars/Main.java (hunk 1 above)
Comment (linusstretchtips) [reply plugin/src/main/java/org/screamingsandals/bedwars/Main.java:556]: EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

**Review by Misat11 — APPROVED** [review]

**Comment by linusstretchtips** [comment]
maybe merge this :lul:

**Pull request merged**